main.go

```diff
@@ -137,16 +100,18 @@ func main() {
 
 	r.GET(containerURL+"/WS", func(c *gin.Context) {
 		id := c.Param("id")
-		if val, ok := containers.Load(id); !ok {
-			infologger.Printf("Channel %s not found. Adding new channel\n", id)
-			container := make(chan types.ContainerJSON)
-			go singleContainerRoutine(id, cli, container)
-			go sendJSONContainerRoutine(m, container, c.Request.URL.Path)
-			containers.Store(id, SubscribedChan{subCount: 1, channel: container})
+		if val, ok := containers[id]; !ok {
+			infoLogger.Printf("Channel %s not found. Adding new containers\n", id)
+			//container := make(chan types.ContainerJSON)
+			//go singleContainerRoutine(id, cli, container)
+			//go sendJSONContainerRoutine(m, container, c.Request.URL.Path)
+			cont := ContainerBroadcaster{subCount: 1, mel: m, logger: debugLogger, cli: cli, quit: make(chan bool)}
```

Comment:
Quit channel created here and in a Start method, this channel will be removed. It is just more work for gc.